lib/compress/zstd_compress.c

```diff
@@ -50,6 +50,7 @@ struct ZSTD_CDict_s {
     ZSTD_compressedBlockState_t cBlockState;
     ZSTD_customMem customMem;
     U32 dictID;
+    BYTE compressionLevel; /* 0 indicates that advanced API was used to select CDict params */
```

Comment:
I guess it should be an `int` instead, to respect the API contract.
`int` can also represent negative values, which are useful in some high speed scenarios.